src/Congo.WebApi/Data/ProductAccess/InsertProductCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Congo.WebApi.Data.Models;
+using MediatR;
+
+namespace Congo.WebApi.Data.ProductAccess
+{
+    public record InsertProductCommand(string Name, string Description, decimal Price, string ImageUrl) :IRequest<Product>;
+    //This needs a Handler
```

Comment:
This comment should be above the command, have a space between the slashes and the text, and probably have a TODO prefix.
 Like so:
```cs
// TODO: this needs a handler
```

Reply:
Fixed formatting